lms/djangoapps/bulk_email/tasks.py

```diff
@@ -381,7 +382,8 @@ def _get_source_address(course_id, course_title):
     # For the email address, get the course.  Then make sure that it can be used
     # in an email address, by substituting a '_' anywhere a non-(ascii, period, or dash)
     # character appears.
-    from_addr = u'"{0}" Course Staff <{1}-{2}>'.format(
+    # Translators: This is the email subject when the course staff emails the students
```

Comment:
This is not the email subject, it is the email's "From" address. I suggest doing the following:
```python
# Translators: This is the name that a sent email is from, for example, "Physics" Course Staff
course_staff_title = _('"{course_name}" Course Staff')
from_addr = course_staff_title + u' <{course_id}-{from_email}>'
from_addr = from_addr.format(
    course_name=course_title_no_quotes,
    course_id=re.sub(r"[^\w.-]", '_', course_id.course),
    from_email=settings.BULK_EMAIL_DEFAULT_FROM_EMAIL
)
```